pkg/apiserver/webhooks/cdi-validate_test.go

```diff
@@ -22,6 +22,8 @@ package webhooks
 import (
 	"encoding/json"
 
+	sdkapi "github.com/kubevirt/controller-lifecycle-operator-sdk/pkg/sdk/api"
```

Comment:
why aren't these imports `kubevirt.io/controller-lifecycle-operator-sdk/pkg/sdk/api`?

Reply:
https://github.com/kubevirt/containerized-data-importer/issues/1380

To be implemented in follow up PR